docs/using/subscriptions.md

```diff
@@ -12,7 +12,7 @@ To create a subscription, users define the following parameters:
   
 As ASF receives new acquisitions that meet the search criteria for the subscription, they will automatically be submitted to HyP3 for On Demand processing using the defined parameters. 
 
```

Comment:
"to search regularly" - originally I read that as meaning to search in a usual fashion.  I think using something like "to periodically search"
Just a suggestion.